test/dummy/config/environment.rb

```diff
@@ -1,5 +1,9 @@
 # Load the rails application
 require File.expand_path('../application', __FILE__)
 
+Rails.application.configure do
+  config.active_support.test_order = :sorted
```

Comment:
It does, but the original default was :sorted - according the warning output that I had, so I didn't want to change things. 

And here, I kind of disagree, since I have a global configuration test in AggregateTest that checks if the value is still set, and the resets, but that test is quick to change since I suspect it's not great practice